Minor warning unrelated to the PR, just for the record:
```
../librz/core/agraph.c: In function ‘set_layout’:
../librz/core/agraph.c:1046:21: warning: ‘dist’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1046 |   dist = is_first? v: RZ_MIN (dist, v);
../librz/core/agraph.c:1031:6: note: ‘dist’ was declared here
 1031 |  int dist, v, is_first = true;
      |      ^~~~
../librz/core/agraph.c:1046:21: warning: ‘dist’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 1046 |   dist = is_first? v: RZ_MIN (dist, v);
../librz/core/agraph.c:1031:6: note: ‘dist’ was declared here
 1031 |  int dist, v, is_first = true;
      |      ^~~~
In file included from ../librz/include/rz_main.h:6,
                 from ../librz/include/rz_core.h:6,
                 from ../librz/core/agraph.c:3:
In function ‘backedge_info’,
    inlined from ‘set_layout’ at ../librz/core/agraph.c:2065:2:
../librz/include/rz_types.h:320:27: warning: argument 1 range [18446744071562067968, 18446744073709551615] exceeds maximum object size 9223372036854775807 [-Walloc-size-larger-than=]
  320 | #define RZ_NEWS0(x,y) (x*)calloc(y,sizeof(x))
      |                           ^~~~~~~~~~~~~~~~~~~
../librz/core/agraph.c:1762:14: note: in expansion of macro ‘RZ_NEWS0’
 1762 |  int **arr = RZ_NEWS0 (int *, g->n_layers);
      |              ^~~~~~~~
In file included from ../librz/include/rz_types.h:243,
                 from ../librz/include/rz_main.h:6,
                 from ../librz/include/rz_core.h:6,
                 from ../librz/core/agraph.c:3:
../librz/core/agraph.c: In function ‘set_layout’:
/usr/include/stdlib.h:542:14: note: in a call to allocation function ‘calloc’ declared here
  542 | extern void *calloc (size_t __nmemb, size_t __size)

In file included from ../librz/bin/format/pe/pe64.c:4:
../librz/bin/format/pe/pe.c: In function ‘Pe64_rz_bin_pe_get_debug_data’:
../librz/bin/format/pe/pe.c:3689:48: warning: ‘img_dbg_dir_entry.PointerToRawData’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 3689 |  ut32 dbg_data_poff = RZ_MIN (img_dbg_dir_entry.PointerToRawData, rz_buf_size (bin->b));
      |                                                ^
../librz/bin/format/pe/pe.c:3690:6: warning: ‘img_dbg_dir_entry.SizeOfData’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 3690 |  int dbg_data_len = RZ_MIN (img_dbg_dir_entry.SizeOfData, rz_buf_size (bin->b) - dbg_data_poff);
      |      ^~~~~~~~~~~~
../librz/bin/format/pe/pe.c:3586:2: warning: ‘img_dbg_dir_entry.Type’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 3586 |  switch (dbg_dir_entry->Type) {
      |  ^~~~~~
../librz/bin/format/pe/pe.c:3669:35: note: ‘img_dbg_dir_entry.Type’ was declared here
 3669 |  PE_(image_debug_directory_entry) img_dbg_dir_entry;
      |                                   ^~~~~~~~~~~~~~~~~
./librz/bin/format/pe/pe.c: In function ‘Pe32_rz_bin_pe_get_debug_data’:
../librz/bin/format/pe/pe.c:3689:48: warning: ‘img_dbg_dir_entry.PointerToRawData’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 3689 |  ut32 dbg_data_poff = RZ_MIN (img_dbg_dir_entry.PointerToRawData, rz_buf_size (bin->b));
      |                                                ^
../librz/bin/format/pe/pe.c:3690:6: warning: ‘img_dbg_dir_entry.SizeOfData’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 3690 |  int dbg_data_len = RZ_MIN (img_dbg_dir_entry.SizeOfData, rz_buf_size (bin->b) - dbg_data_poff);
      |      ^~~~~~~~~~~~
../librz/bin/format/pe/pe.c:3586:2: warning: ‘img_dbg_dir_entry.Type’ may be used uninitialized in this function [-Wmaybe-uninitialized]
 3586 |  switch (dbg_dir_entry->Type) {
      |  ^~~~~~
../librz/bin/format/pe/pe.c:3669:35: note: ‘img_dbg_dir_entry.Type’ was declared here
 3669 |  PE_(image_debug_directory_entry) img_dbg_dir_entry;
      |                                   ^~~~~~~~~~~~~~~~~
[607/1491] Compiling C object librz/bin/librz_bin.a.p/format_wasm_wasm.c.o
../librz/bin/format/wasm/wasm.c: In function ‘consume_s7_r’:
../librz/bin/format/wasm/wasm.c:63:9: warning: ‘n’ may be used uninitialized in this function [-Wmaybe-uninitialized]
   63 |  return n;
      |         ^
[655/1491] Compiling C object librz/lang/librz_lang.a.p/lang.c.o
In file included from ../librz/include/rz_cons.h:19,
                 from ../librz/include/rz_diff.h:6,
                 from ../librz/include/rz_util.h:7,
                 from ../librz/lang/lang.c:4:
../librz/lang/p/pipe.c: In function ‘lang_pipe_run’:
../librz/include/rz_util/rz_sys.h:80:23: warning: ignoring return value of ‘system’, declared with attribute warn_unused_result [-Wunused-result]
   80 | #define rz_sys_system system
../librz/lang/p/pipe.c:167:3: note: in expansion of macro ‘rz_sys_system’
  167 |   rz_sys_system (code);
      |   ^~~~~~~~~~~~~
In file included from ../librz/lang/lang.c:8:
../librz/lang/p/pipe.c:168:10: warning: ignoring return value of ‘write’, declared with attribute warn_unused_result [-Wunused-result]
  168 |   (void) write (input[1], "", 1);
      |          ^~~~~~~~~~~~~~~~~~~~~~~
../librz/lang/p/pipe.c:202:12: warning: ignoring return value of ‘write’, declared with attribute warn_unused_result [-Wunused-result]
  202 |     (void) write (input[1], res, strlen (res) + 1);
      |            ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
../librz/lang/p/pipe.c:206:12: warning: ignoring return value of ‘write’, declared with attribute warn_unused_result [-Wunused-result]
  206 |     (void) write (input[1], "", 1); // NULL byte
      |            ^~~~~~~~~~~~~~~~~~~~~~~
In file included from ../librz/include/rz_cons.h:19,
                 from ../librz/include/rz_diff.h:6,
                 from ../librz/include/rz_util.h:7,
                 from ../librz/lang/lang.c:4:
../librz/lang/lang.c: In function ‘rz_lang_prompt’:
../librz/include/rz_util/rz_sys.h:80:23: warning: ignoring return value of ‘system’, declared with attribute warn_unused_result [-Wunused-result]
   80 | #define rz_sys_system system
../librz/lang/lang.c:278:5: note: in expansion of macro ‘rz_sys_system’
  278 |     rz_sys_system (buf + 1);
      |     ^~~~~~~~~~~~~
[805/1491] Compiling C object librz/asm/librz_asm.a.p/arch_tms320_c55x_plus_c55plus_decode.c.o
In file included from /usr/include/stdio.h:867,
                 from ../librz/asm/arch/tms320/c55x_plus/c55plus_decode.c:3:
In function ‘fprintf’,
    inlined from ‘decode_ins’ at ../librz/asm/arch/tms320/c55x_plus/c55plus_decode.c:238:3:
/usr/include/x86_64-linux-gnu/bits/stdio2.h:100:10: warning: ‘%s’ directive argument is null [-Wformat-overflow=]
  100 |   return __fprintf_chk (__stream, __USE_FORTIFY_LEVEL - 1, __fmt,
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  101 |    __va_arg_pack ());
      |    ~~~~~~~~~~~~~~~~~
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -381,6 +381,7 @@ jobs:
       run: |
         export PATH=${HOME}/.local/bin:${PATH}
         ninja -C build test
+      working-directory: rizin
     - name: Run tests
       run: |
         export PATH=${HOME}/.local/bin:${PATH}

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -372,6 +372,7 @@ jobs:
         git submodule update
     - name: Checkout our Testsuite Binaries
       run: git clone https://github.com/rizinorg/rizin-testbins test/bins
+      working-directory: rizin
     - name: Compile with meson
       run: |
         CFLAGS="-static" LDFLAGS="-static" meson --prefix=${HOME}/.local --buildtype release --default-library static build

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -358,31 +358,42 @@ jobs:
     continue-on-error: true
     if: contains(github.head_ref, 'dist') || contains(github.head_ref, 'static') || contains(github.ref, 'release-') || github.event_name == 'schedule'
     runs-on: ubuntu-20.04
+    container: alpine:edge
     steps:
-    - uses: actions/checkout@v2
-      with:
-          submodules: recursive
-    - name: Checkout our Testsuite Binaries
-      uses: actions/checkout@v2
-      with:
-          repository: rizinorg/rizin-testbins
-          path: test/bins
-    - name: Install static
+    - name: Install dependencies
+      run: apk add --no-cache git gcc g++ linux-headers cmake wget unzip py3-wheel py3-setuptools py3-pip meson ninja
+    - name: Checkout rizin
       run: |
-        ./sys/static.sh
-        sudo make symstall
+        git clone https://github.com/${{ github.repository }}
+        cd rizin
+        git fetch origin ${{ github.ref }}
+        git checkout -b local_branch FETCH_HEAD
+        git submodule init
+        git submodule update
+    - name: Checkout our Testsuite Binaries
+      run: git clone https://github.com/rizinorg/rizin-testbins test/bins
+    - name: Compile with meson
+      run: |
+        CFLAGS="-static" LDFLAGS="-static" meson --prefix=${HOME}/.local --buildtype release --default-library static build
+        ninja -C build && ninja -C build install
+      working-directory: rizin
+    - name: Run unit tests
+      run: |
+        export PATH=${HOME}/.local/bin:${PATH}
+        ninja -C build test
     - name: Run tests
       run: |
+        export PATH=${HOME}/.local/bin:${PATH}
         rizin -v
         rz-test -v
         cd test
-        make
+        rz-test -L -o results.json
+      working-directory: rizin
     - name: Upload test results
       uses: actions/upload-artifact@v2
       with:
         name: test-results-static
         path: test/results.json
-
 
   create-tarball:
     name: Create source tarball

